src/partition.rs

```diff
@@ -177,3 +184,12 @@ pub enum TxSetType {
     EXT_TX_SET_ALL16 = 5,
     EXT_TX_SET_TYPES
 }
+
+pub fn get_subsize(bsize: BlockSize , partition: PartitionType) -> BlockSize {
+    if partition == PartitionType::PARTITION_INVALID {
```

Comment:
Do we need PARTITION_INVALID at all? Is there any way the current code can make it?

Reply:
Good point. There is no chance that current code goes that path.
In more detail, I just learned that it is only used by libaom when final bitstream is written. 
That is libaom save all coding decisions for all the tiles in a frame, then later pack_stream() write those coding decisions to bitstream. By doing so, write_modes_sb() fetches the partition decision thru get_partition(cm, mi_row, mi_col, bsize) then it probably wants to prepare for a mi_row, mi_col with partition type not set at all.

So, ok- let's remove this copy of legacy code.